docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,231 @@
+working at scale, set `renormalization_window_days` to a lower value, so the 
+workload is reduced.
+
+
+[[model-snapshots]]
+=== Setting model snapshot retention time 
+
+Model snapshots are created every 3 to 4 hours by default. Although the 
+snapshots are compressed, they can still be extensive for large jobs. By 
+default, if there are snapshots over one day older than the newest snapshot, 
+they are deleted except for the first snapshot each day. As well, all snapshots 
+over ten days older than the newest snapshot are deleted. There are two things 
+to consider about snapshot retention:
+
+* How frequently you want to create snapshots from the model
+* How long you want to retain the snapshots.
+
+You can fine-tune the frequency of the model persistence and the snapshot 
+retention days by setting the following configuration options:
+
+* `background_persist_interval` sets the time between each periodic persistence 
+  of the model. The default value is a randomized value between 3 to 4 hours, 
+  which avoids all jobs persisting at the same time. Increase this value 
+  appropriately to fit your job to reduce the work load. Note that a higher 
+  value means that the job may restore older model state in the event of a node 
+  failure.
+
+* `daily_model_snapshot_retention_after_days` affects the automatic removal of 
+  old model snapshots for the job. It specifies a period of time (in days) after 
+  which only the first snapshot per day is retained. This period is relative to 
+  the timestamp of the most recent snapshot for this job. Consider setting this 
+  value to only a couple of days after which one snapshot per day will be 
+  retained. Jobs created in 7.8 and above have this value set to 1. Consider 
+  setting this to 1 for jobs created prior to 7.8.
+
+* `model_snapshot_retention_days` defines the time in days that model snapshots 
+  are retained for the job. The default value is ten days meaning snapshots ten 
+  days older than the newest snapshot are deleted. A smaller value reduces the 
+  size of the model state stored on disk, but the time window from which the 
+  model can be restored is also smaller. 
+
+For more information, refer to <<ml-model-snapshots>>.
+
+TIP: When your job is shut down, and it is not closed properly, it rolls back to 
+the last successfully saved snapshot when it is started again. Close the job 
+gracefully when it is possible to preserve the exact model state.
+
+
+[[short-bucket-span]]
+=== Avoiding short bucket span for large jobs
+
+For machine data, bucket span is typically between 15m and 1h. The recommended 
+value always depends on the data, the use case, and the latency required for 
+alerting. If you have a large amount of data, use longer bucket span because 
+less resources are required for processing each bucket and fewer results are 
+written. Bucket spans that are sensible dividers of an hour / day work best as 
+most periodic patterns have a daily cycle.
```

Comment:
Can we please move short bucket spans further up? After "choosing influencers".

Also please change from 

```
If you have a large amount of data, use longer bucket span because 
less resources are required for processing each bucket and fewer results are 
written.
```

to

```
A job with a longer bucket span will use less resources 
because fewer buckets require processing and fewer results are written.
```

This seems picky, however the information is true regardless of the size of the data. As you say in the following para, it relevant where short buckets and high cardinality clash.